packages/components/src/SVG/icons/ChevronDownIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { SVG, SVGProps } from "~components/SVG"
+
+export const ChevronDownIcon = (
```

Comment:
Attempted the Solution Preview which didn't work, so going with this internally for now.